content/wiki/canonical_lints.md

```diff
@@ -68,15 +71,31 @@ clippy.wildcard_dependencies = "warn"
 # END LINEBENDER LINT SET
 ```
 
-And in their `lib.rs`:
+## `lib.rs`
 
 ```rust
-// LINEBENDER LINT SET - v1
+// LINEBENDER LINT SET - lib.rs - v1
 // See https://linebender.org/wiki/canonical-lints/
 // These lints aren't included in Cargo.toml because they
 // shouldn't apply to examples and tests
 #![warn(unused_crate_dependencies)]
 #![warn(clippy::print_stdout, clippy::print_stderr)]
+// END LINEBENDER LINT SET
```

Comment:
Previous discussion here https://github.com/linebender/linebender.github.io/pull/68#discussion_r1803688665

Reply:
The rustfmt problem is valid, but looking at [this in the wild](https://github.com/linebender/xilem/blob/d8d916c2d8432374e8c06b8d297a674472c5e86e/xilem/src/lib.rs#L6-L39) it's not immediately obvious where the set ends. I'll need to think about this a bit more.

Reply:
I like it, as we want that present everywhere anyway.